module/execution/scripts.go

```diff
@@ -42,15 +42,25 @@ type ScriptExecutor interface {
 	// - storage.ErrHeightNotIndexed if the data for the block height is not available
 	GetAccountAtBlockHeight(ctx context.Context, address flow.Address, height uint64) (*flow.Account, error)
 
-	// GetAccountBalance returns
+	// GetAccountBalance returns a Flow account balance by the provided address and block height.
 	// Expected errors:
 	// - storage.ErrHeightNotIndexed if the data for the block height is not available
 	GetAccountBalance(ctx context.Context, address flow.Address, height uint64) (uint64, error)
 
-	// GetAccountKeys returns
+	// GetAccountAvailableBalance returns a Flow account available balance by the provided address and block height.
+	// Expected errors:
+	// - storage.ErrHeightNotIndexed if the data for the block height is not available
+	GetAccountAvailableBalance(ctx context.Context, address flow.Address, height uint64) (uint64, error)
+
+	// GetAccountKeys returns a Flow account public keys by the provided address and block height.
 	// Expected errors:
 	// - storage.ErrHeightNotIndexed if the data for the block height is not available
 	GetAccountKeys(ctx context.Context, address flow.Address, height uint64) ([]flow.AccountPublicKey, error)
+
+	// GetAccountKey returns a Flow account public key by the provided address, block height and index.
+	// Expected errors:
+	// - storage.ErrHeightNotIndexed if the data for the block height is not available
+	GetAccountKey(ctx context.Context, address flow.Address, keyIndex int, height uint64) (*flow.AccountPublicKey, error)
```

Comment:
key index is a `uint64` most places in flow-go. let's just use that everywhere to avoid weird value truncation bugs in the future
```suggestion
	GetAccountKey(ctx context.Context, address flow.Address, keyIndex uint64, height uint64) (*flow.AccountPublicKey, error)
```